package.json

```diff
@@ -29,13 +29,18 @@
   ],
   "author": "",
   "license": "MIT",
+  "dependencies": {
+    "fast-json-patch": "github:wet-boew/JSON-Patch#wb1.0+1.1.4",
+    "jsonpointer.js": "^0.4.0",
+    "wet-boew": "LaurentGoderre/wet-boew#themes-npm"
+  },
   "devDependencies": {
-    "assemble-contrib-i18n": "~0.1.2",
+    "assemble-contrib-i18n": "0.1.*",
```

Comment:
Not sure I get this change. Wouldn't these be equivalant, or is the `~` a problem because it hasn't hit a major version yet?

Reply:
I'm just making it consistent with WET core